### PR TITLE
Use HTTPS whenever possible

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -40,7 +40,7 @@ your system:
 
 .. code-block:: bash
 
-    $ wget http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar -O php-cs-fixer
+    $ wget https://cs.sensiolabs.org/download/php-cs-fixer-v2.phar -O php-cs-fixer
```
Comment (fabpot) [inline README.rst]: We might not want to use HTTPS. WDYT?

Comment (julienfalque) [reply README.rst]: Why would we want to _not_ use HTTPS?

**Comment by SpacePossum** [comment]
Thanks, awesome :) 
Great to see the releases on `cs.sensiolabs.org` can be fetch over HTTPS.

Practical question at the maintainers, do we want the changes on `2.2` as well (`1.13` still exists as branch but is EOL so doesn't matter to much I think)?

**Comment by julienfalque** [comment]
I'd say 2.2 for the README changes and 2.11 for `TypeAnalysis` as it does not exist in branch 2.2.

File: README.rst (hunk 1 above)
Comment (keradus) [reply README.rst]: well, we wanted HTTPS since beginning, simply https wasn't working properly on this subdomain originally

**Comment by keradus** [comment]
thanks for pointing this out @fabpot !

as a side effect, you can see your PR is failing on fabbot, which is asking to use long-array syntax, even if we have config to use short syntax. Can you take a look?

File: README.rst (hunk 1 above)
Comment (julienfalque) [reply README.rst]: I know, but now it does and I see no reason not to use it, hence my question :)

File: src/Tokenizer/Analyzer/Analysis/TypeAnalysis.php [hunk 2]
```diff
@@ -22,9 +22,9 @@
      *
      * More info:
      *
-     * @see http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration.types
-     * @see http://php.net/manual/en/reserved.other-reserved-words.php
-     * @see http://php.net/manual/en/language.pseudo-types.php
+     * @see https://secure.php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration.types
```
Comment (carusogabriel) [inline src/Tokenizer/Analyzer/Analysis/TypeAnalysis.php]: We can remove the `secure` subdomain

**Comment by keradus** [comment]
Thank you @fabpot.

**Pull request closed without merging**